jekyll/_cci2_ja/nomad.adoc

```diff
@@ -22,13 +22,13 @@ toc::[]
 .Nomad クラスタの管理
 image::nomad-diagram-v2.png[Diagram of the Nomad cluster]
 <<<
-- **Nomad サーバー:** Nomad サーバーは、Nomad クラスタの "頭脳" です。 ジョブを受け取り、Nomad クライアントに割り当てます。 CircleCI では、Nomad サーバーは Docker コンテナとしてサービスマシンで実行されます。
+- *Nomad サーバー:* Nomad サーバーは、Nomad クラスタの "頭脳" です。 ジョブを受け取り、Nomad クライアントに割り当てます。 CircleCI では、Nomad サーバーは Docker コンテナとしてサービスマシンで実行されます。
```

Comment:
Thank you! I concerned the same thing and confirmed it locally. It kept bold. I agree it's not ideal but would like to leave this as it is since Crowdin treats bold as one asterisk (which means I have to do this conversion every time manually) 🙏 
![スクリーンショット 2022-05-25 6 08 06](https://user-images.githubusercontent.com/87592312/170132628-1e91ea5b-4604-41d3-8181-dc48397c51a2.png)

![スクリーンショット 2022-05-25 6 05 14](https://user-images.githubusercontent.com/87592312/170132289-98654791-2e71-471c-bc85-9c47d87d33d6.png)

I hope this issue will be resolved in the next translation management system!